dtmi/wurthelektronik/desigkkit/calypsoiotkit-1.json

```diff
@@ -0,0 +1,165 @@
+{
+	"@context": [
+		"dtmi:iotcentral:context;2",
+		"dtmi:dtdl:context;2"
+	],
+	"@id": "dtmi:wurthelektronik:designkit:calypsoiotkit;1",
```

Comment:
there is a typo in the file name.. I'd suggest to use `dmr-client` to avoid these issues

Reply:
dmr-client check done. But, cannot test the kit with this tool as we do not have the component (Calypso Wi-Fi module) in the public repository yet.